tests/languages/python/decorator_feature.test

```diff
@@ -0,0 +1,28 @@
+@some.decorator(foobar)
```

Comment:
Do we want to highlight the whole line as a decorator or just the `@` symbol? How do other highlighters do it?

Reply:
I err towards highlighting `@some.decorator` as Notepad++ and VS both do.
(VS Code doesn't highlight anything)